File: src/targets/csharp/httpclient/fixtures/nested.cs
using System.Net.Http.Headers;
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Get,
    RequestUri = new Uri("http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value"),
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: src/targets/javascript/jquery/fixtures/nested.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'http://mockbin.com/har?foo[bar]=baz,zap&fiz=buz&key=value',
  method: 'GET',
  headers: {}
};

$.ajax(settings).done(function (response) {
  console.log(response);
});